charts/osm/values.schema.json

```diff
@@ -487,6 +486,100 @@
                     "examples": [
                         false
                     ]
+                },
+                "injector": {
+                    "$id": "#/properties/OpenServiceMesh/properties/injector",
+                    "type": "object",
+                    "title": "The sidecar injector schema",
+                    "description": "Sidecar injector configurations",
+                    "examples": [
+                        {
+                            "enable": true
+                        }
+                    ],
```

Comment:
I don't see an `enable` field anywhere having to do with the injector. Could we make this either a realistic example or skip it entirely?

Reply:
Done.